examples/js/exporters/GLTFExporter.js

```diff
@@ -180,6 +180,27 @@ THREE.GLTFExporter.prototype = {
 
 		}
 
+		/**
+		 * Get the required size + padding for a buffer, rounded to the next 4-byte boundary.
+		 * https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#data-alignment
+		 *
+		 * @param {Integer} bufferSize The size the original buffer.
+		 * @returns {Integer} new buffer size with required padding.
+		 *
+		 */
+		function getPaddedBufferSize( bufferSize ) {
+			
+			// adapted from https://github.com/AnalyticalGraphicsInc/obj2gltf/blob/master/lib/getBufferPadded.js
+
+			var boundary = 4;
+
+			var remainder = bufferSize % boundary;
+			
+			var padding =  remainder === 0 ? 0 : boundary - remainder;
+
+			return bufferSize + padding;
```

Comment:
How about
```javascript
function getPaddedBufferSize( bufferSize ) {

	return Math.ceil( bufferSize / 4 ) * 4;

}
```

Reply:
I'd have to leave that for debate. Your example is wonderfully succinct but the original is easier to read.

Reply:
I'll let @donmccurdy decide 😇

Reply:
The code you copied pads a buffer. You just need to calculate a length.

Reply:
Indeed, we do need to calculate a length - for a padded buffer. I appreciate the terseness of your suggested alternative from a nerdy point of view, but not regards readability. I'm not massively fussed which approach is accepted however I'm just knocking down gltf validation errors.

Reply:
Yeah, I think +1 for the short version. There are already plenty of comments explaining the purpose clearly.

Reply:
Ok, done!